rpmlint/descriptions/SUIDPermissionsCheck.toml

```diff
@@ -0,0 +1,40 @@
+AUDIT_BUG_URL = "https://en.opensuse.org/openSUSE:Package_security_guidelines#audit_bugs"
+REVIEW_NEEDED_TEXT = "If the package is intended for inclusion in any SUSE product please open a bug report to request review of the package by the security team. Please refer to #AUDIT_BUG_URL# for more information."
+GHOST_ENCOUNTERED_TEXT = "This is not allowed, since it is impossible to review. Please refer to #AUDIT_BUG_URL# for more information."
+
+permissions-unauthorized-file="""#REVIEW_NEEDED_TEXT#"""
+permissions-symlink="""
+permissions handling for symlinks is useless. Please contact security@suse.de to remove the entry. Please refer to #AUDIT_BUG_URL# for more information.
+"""
+permissions-dir-without-slash="""
+the entry in the permissions file refers to a directory. Please contact security@suse.de to append a slash to the entry in order to avoid security problems. Please refer to #AUDIT_BUG_URL# for more information.
+"""
+permissions-file-as-dir="""
+the entry in the permissions file refers to a directory but the package actually contains a file. Please contact security@suse.de to remove the slash. Please refer to #AUDIT_BUG_URL# for more information.
+"""
+permissions-incorrect="""
+please use the %attr macro to set the correct permissions.
+"""
+permissions-incorrect-owner="""
+please use the %attr macro to set the correct ownership.
+"""
+permissions-file-setuid-bit="""#REVIEW_NEEDED_TEXT#"""
+permissions-directory-setuid-bit="""#REVIEW_NEEDED_TEXT#"""
```

Comment:
For `file-setuid-bit` and `directory-setuid-bit` the cause for the limitation is missing from the description. In the original check we have this text:

```
%(file)s is packaged with ' \
                          'setuid/setgid bits (0%(mode)o)
```

Reply:
This happens in L27 of the SUIDPermissionsCheck.py file, should I put it in here? It could be hard because of the params.
The output currently is being printed and can be verified with the `test/binary/testsuidpermissionscheck-0.0.1-0.x86_64.rpm` binary

Reply:
Okay I missed that. As long as the end user in OBS will get an appropriate explanation then it should be fine.